.github/renovate.json5

```diff
@@ -31,7 +31,7 @@
     "typescript", // >4.5.x doesn't seem to work on windows with pnp currently. probably similar to https://github.com/yarnpkg/berry/issues/3722
     "ora" // we should bump this once we move to esm modules
   ],
-  "ignorePaths": ["packages/grafana-toolkit/package.json", "emails/**", "plugins-bundled/**", "**/mocks/**"],
```

Comment:
are you sure about this? 😁 

i mean, isn't it better to only do this upgrade-thing before every major-grafana-release or some other, very controlled way?

Reply:
Yeah you are right @gabor . I guess we want patch releases to go into toolkit deps as that would likely help reduce the number of security warnings. I'll poke around the renovate docs to see what is possible.